views/src/app/home/shared/home.service.ts

```diff
@@ -98,4 +98,9 @@ export class HomeService {
         }))
           .catch((error:any) => Observable.throw(error.json().error || 'Server error'));
     }
+    deleteSession() {
```

Comment:
I still see the following issues: 
1. Upon logging out, the session is not deleted.
2. Upon Remove service operation, the session is not deleted.
3. When the browser is closed, the session is not deleted. 
4. The delete icon on the session is no longer seen to invoke the delete operation manually.

I did login with the newer credentials successfully.

Reply:
I am also seeing the delete issue.